resources/assets/components/InboxItem/index.js

```diff
@@ -39,6 +41,22 @@ class InboxItem extends React.Component {
     const campaign = this.props.details.campaign;
     const signup = this.props.details.signup;
 
+    if (post['user']) {
```

Comment:
@DFurnes the first scenario with `post['user']` is for populating the inbox items for the Campaign Inbox page. It looks like there is a TODO [here](https://github.com/DoSomething/rogue/blob/master/app/Http/Controllers/CampaignsController.php#L64-L75) suggesting to grab the user with the signup which would actually solve this problem and then we wouldn't have to do a conditional! Should I work on that or is that outside of the scope of this PR?